pkgs/by-name/li/libgudev/package.nix

```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation (finalAttrs: {
     umockdev
   ];
 
-  doCheck = withIntrospection;
+  doCheck = withIntrospection && (lib.hasPrefix "systemd" udev.name);
```

Comment:
Does this package actually work with any of the non-systemd udev implementations we have? I vaguely recall concluding that it didn't when I looked at this before, but it was long ago and I could be wrong.